c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/Train.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace c_sharp_apps_shay_shalom_izhak.transportaion_Cargo_App
+{
+    public class Train : CargoVehicle
+    {
+        private Cron cargoLimiter;
+
```

Comment:
I am expecting to see here array or list not single Cron (like in the real world). 
At ship too - multiple Containers.